<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login setup</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Login setup</string>
   </property>
   <widget class="QLabel" name="avatarLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>92</y>
      <width>42</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Avatar:</string>
    </property>
   </widget>
   <widget class="QLabel" name="avatarPixmap">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>118</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="changeAvatarButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Change avatar</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>17</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name:</string>
       </property>
       <property name="buddy">
        <cstring>nameLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Password:</string>
       </property>
       <property name="buddy">
        <cstring>passwordLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="passwordLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QListWidget" name="avatarList">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>481</width>
     <height>261</height>
    </rect>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="movement">
    <enum>QListView::Static</enum>
   </property>
   <property name="viewMode">
    <enum>QListView::IconMode</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>avatarList</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
